app/views/layouts/_header.html.erb

```diff
@@ -35,8 +35,12 @@
                 </li>
                 <li><%= link_to "Kontakt", kontakt_path %>
                 </li>
-                <li><%= link_to "Admin (pojawi sie jezeli zalogowany)" %>
-                </li>
+                <% if user_signed_in? %>
+                    <li><%= link_to "Panel Admina", rails_admin_path %>
```

Comment:
"Panel Admina" should be always visible, even for not logged user. In case that not logged user click on it, it should redirect to /sign_in